### PR TITLE
Feature/lpfg 394 payment methods

### PR DESCRIPTION
**Pull request by superafroman**

### CODE REVIEW

File: src/main/resources/db/migration/mysql/V1.5.0__add-payment-method.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+CREATE TABLE `department_payment_methods` (
+  `department_id` smallint(5) DEFAULT NOT NULL,
```
Comment (elxsy) [inline src/main/resources/db/migration/mysql/V1.5.0__add-payment-method.sql]: OPTIMISATION - unsigned integers please to save space and increase range.

Comment (superafroman) [reply src/main/resources/db/migration/mysql/V1.5.0__add-payment-method.sql]: This is a foreign key onto the departments table so needs to be the same data type. If you want that changes please raise a task and we'll do it outside of this story.

Comment (elxsy) [reply src/main/resources/db/migration/mysql/V1.5.0__add-payment-method.sql]: Max, data type is absolutely fine. Sign needs to be dropped off only.

https://github.com/Civil-Service-Human-Resources/civil-servant-registry-service/blob/master/src/main/resources/db/migration/mysql/V1__init.sql#L56

`... smallint(5) unsigned NOT NULL`

Comment (superafroman) [reply src/main/resources/db/migration/mysql/V1.5.0__add-payment-method.sql]: Ah - was looking at H2. Updated.

**Pull request merged**